### PR TITLE
feat(api): get user's connected providers

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#346

Ticket: 346

### Dependencies

None

### Description

End point for connected providers

### Release Plan

Merge once approved

### CODE REVIEW

File: packages/packages/api/src/client/metriport.ts [hunk 1]
```diff
@@ -41,6 +41,18 @@ export class Metriport {
...
   }
 
+  /**
+   * For your given user ID, returns the user's connected providers
+   *
+   * @param {string} userId - The unique ID for the user in your app.
+   * @returns Array of connected providers
+   */
+  async getConnectedProviders(userId: string): Promise<string[]> {
+    const resp = await this.api.get<string[]>(`/user/${userId}/connect`);
```
Comment (Goncharo) [inline packages/packages/api/src/client/metriport.ts:51]: needs updating:
- `@returns Object containing array of connected providers.`
- `Promise<{connectedProviders: string[]}>` 
- `/user/${userId}/connected-providers`

File: docs/user/get-connected-providers.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+</ResponseField>
+
+<ResponseExample>
+
+```javascript Metriport SDK
+import { Metriport } from "@metriport/api";
+
+const metriportClient = new Metriport("YOUR_API_KEY");
+
+const metriportUserId = await metriportClient.getConnectedProviders("metriportUserId");
```
Comment (Goncharo) [inline docs/user/get-connected-providers.mdx:20]: `const connectedProviders = ...`

**Review by Goncharo — CHANGES_REQUESTED** [review]

**Review by leite08 — APPROVED** [review]
Just what Dima pointed out, other than that looks good! :shipit:

**Review by Goncharo — APPROVED** [review]
🚢 🚢 🚢

**Pull request merged**